### PR TITLE
Close the milestones on a release

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/developer/build_release.rst [hunk 1]
```diff
@@ -212,4 +212,8 @@ When a new release or a new version is done you should do the following tasks:
     sudo -u sigdev make -f demo.mk build
 
 * Test the demo.
-* Sent a release email to the ``geomapfish@googlegroups.com`` and ``geospatial@lists.camptocamp.com`` mailing lists.
+* Move all the open issue to a new milestone and close the concern milestone
```
Comment (elemoine) [inline doc/developer/build_release.rst]: issue -> issues

Comment (elemoine) [inline doc/developer/build_release.rst]: the concern milestone -> the current milestone

File: doc/developer/build_release.rst [hunk 2]
```diff
@@ -212,4 +212,8 @@ When a new release or a new version is done you should do the following tasks:
...
+  in `cgxp <https://github.com/camptocamp/cgxp/milestones>`
+  and in `c2cgeoportal <https://github.com/camptocamp/c2cgeoportal/milestones>`.
+* Sent a release email to the ``geomapfish@googlegroups.com``
```
Comment (elemoine) [inline doc/developer/build_release.rst]: Sent -> Send

**Comment by elemoine** [comment]
Feel free to merge when my comments are addressed.

**Comment by sbrunner** [comment]
Thanks @elemoine :-)

**Pull request merged**